R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
I'd argue it's still 24 hours worth of information, although maybe not all within a given 'date'. Whether they give hourly data from day 0 at time 01:00, 02:00, 03:00, ... , 24:00 or time 00:00, 01:00, ..., 23:00, both cases are 24 data points and I think 24 hours worth of information - some of that information in the second case can from the previous day contributing to the first data point (day -1 at 23:00 to day 0 at 00:00), however, I think it's still 24 hours worth of information (just shifted over slightly). If we don't think this is 24 hours worth of information, then we should change how metab loops through timeseries data with datetime supplied as it will run metabolism models using data points 00:00 - 23:00 for a given day, not 00:00 - 24:00.